README.md

```diff
@@ -204,28 +204,22 @@ The following are tutorials on KubeLinter written by users. If you have one that
 * [How to add a new check in KubeLinter?](https://www.psaggu.com/upstream-contribution/2021/08/17/notes.html) by Priyanka Saggu.
 * [Extending kube-linter To Build A Custom Template](https://github.com/garethahealy/kubelinter-extending-blog) by [Gareth Healy](https://github.com/garethahealy).
 
-## Community
-
-If you would like to engage with the KubeLinter community, including maintainers and other users, you can join the Slack workspace [here](https://join.slack.com/t/kube-linter/shared_invite/zt-kla9qvyo-Tk~wynTSbr9EE3AjHcv4BQ).
-
-To contribute, check out our [contributing guide](./CONTRIBUTING.md).
-
-As a reminder, all participation in the KubeLinter community is governed by our [code of conduct](./CODE_OF_CONDUCT.md).
+## LICENSE
 
-## WARNING: Alpha release
+KubeLinter is licensed under the [Apache License 2.0](./LICENSE).
 
-KubeLinter is at an early stage of development. There may be breaking changes in
-the future to the command usage, flags, and configuration file formats. However,
-we encourage you to use KubeLinter to test your environment YAML files, see what
-breaks, and [contribute](./CONTRIBUTING.md).
+## Community
 
-## LICENSE
+If you would like to engage with the KubeLinter community, including maintainers and other users, you can join the Slack workspace [here](https://join.slack.com/t/kube-linter/shared_invite/zt-kla9qvyo-Tk~wynTSbr9EE3AjHcv4BQ).
```

Comment:
Maybe we should remove that and keep it GitHub centric.

Reply:
You mean not mentioning slack at all? It's a common practice, for example, [trivy](https://github.com/aquasecurity/trivy), [falco](https://github.com/falcosecurity/falco) do that.